carma/launch/carma_src.launch.py

```diff
@@ -116,6 +116,22 @@ def generate_launch_description():
         default_value= 'False',
         description='Flag to enable opening http tunnesl to CARMA Cloud'
     )
+
+    # Declare system_architecture
+    system_architecture = LaunchConfiguration('system_architecture')
+    declare_system_architecture = DeclareLaunchArgument(
+        name = 'system_architecture',
+        default_value = 'single',
+        description = 'Flag to define whether a single compute system or a dual compute system is being used'
```

Comment:
Can you also specify the accepted values for this config in the description?

Reply:
Technically the accepted values are shown in both of these but I can see how it's not clear. It'd be better to be bolded but can't have that, do you think putting it as 'single' 'dual' / 'manager' 'worker' with single quotes in the text be clear enough?

Reply:
Yes, i think having them in single quotes would make it clear that those are the values we can use here